app/models/api/v3/context_node_type_property.rb

```diff
@@ -42,6 +43,7 @@ class ContextNodeTypeProperty < YellowTable
 
       belongs_to :context_node_type
 
+      validates :prefix, presence: true
```

Comment:
I think we could relax that to only when "role" is present (otherwise node types do not appear as tabs in the dashboards filter panel)